services/workflows-service/src/auth/auth.service.ts

```diff
@@ -11,20 +12,46 @@ export class AuthService {
     private readonly passwordService: PasswordService,
   ) {}
 
+  async validateUserById(userId: string): Promise<UserInfo | null> {
```

Comment:
This function dosent actually validate anything, it takes fetches user data and returns return some of the data